components/05-mis/kubelet-identity.tf

```
@@ -0,0 +1,15 @@
+resource "azurerm_user_assigned_identity" "kubelet_uami" {
+  name = "aks-kubelet-${var.environment}-mi"
+
+  resource_group_name = data.azurerm_resource_group.genesis_rg.name
+  location            = data.azurerm_resource_group.genesis_rg.location
+  tags                = local.common_tags
+}
+
+resource "azurerm_role_assignment" "sbox_registry_acrpull" {
+  for_each             = local.is_sbox ? toset(var.clusters) : toset([])
```

maybe? or just use count?
```suggestion
  for_each = local.is_sbox ? toset(["sbox"]) : toset([])
```